hbase-server/src/main/java/org/apache/hadoop/hbase/replication/regionserver/ReplicationSource.java

```diff
@@ -204,8 +202,6 @@ public void init(Configuration conf, FileSystem fs, ReplicationSourceManager man
     this.waitOnEndpointSeconds =
       this.conf.getInt(WAIT_ON_ENDPOINT_SECONDS, DEFAULT_WAIT_ON_ENDPOINT_SECONDS);
     decorateConf();
-    // 1 second
-    this.sleepForRetries = this.conf.getLong("replication.source.sleepforretries", 1000);
```

Comment:
nit: do you think this `sleepForRetries` would be changing dynamically after the initialization ? if not, can we keep this variable and just call `getSleepForRetries()` once?